itest/integration_test.go

```diff
@@ -119,7 +119,7 @@ func TestTaprootAssetsDaemon(t *testing.T) {
 			// assure no state is taken over between runs.
 			tapdHarness, uniHarness, proofCourier := setupHarnesses(
 				t1, ht, lndHarness, uniServerLndHarness,
-				testCase.proofCourierType,
+				testCase.proofCourierType, testCase.tapdOptions...,
```

Comment:
nit: long line

Reply:
Isn't this line less than 80 chars? What's the rule here? I added a line break